libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx

```diff
@@ -120,42 +125,47 @@ export class ModelOverview extends React.Component<
       });
 
     return (
-      <Stack tokens={{ childrenGap: "10px", padding: "16px 40px 10px 40px" }}>
+      <Stack
+        className={classNames.sectionStack}
+        tokens={{ childrenGap: "10px" }}
+      >
         <Text variant="medium" style={{ maxWidth: descriptionMaxWidth }}>
           {localization.Interpret.ModelPerformance.helperText}
         </Text>
         {!this.props.showNewModelOverviewExperience && <OverallMetricChart />}
         {this.props.showNewModelOverviewExperience && (
           <>
             <Stack horizontal tokens={{ childrenGap: "20px" }}>
-              <Dropdown
+              <ComboBox
```

Comment:
What happens when you choose a ton of features, say 100 features - is the first column still readable in that case?  Does the UI break in any way?

Reply:
Or do we impose some limit on the number of features the user can choose in the dropdown?

Reply:
This is the metrics dropdown. You can choose as many as you like, but there are only like 7 for binary classification that we currently support. Perhaps more in the future.

For the feature dropdown you can only choose two, at which point the other options get disabled. You can see the video for an example (see PR description).